### PR TITLE
minor improvements

### PR DESCRIPTION
**Pull request by maboiteaspam**
enhance example
fix readme syntax
add a README section to help start with library

### CODE REVIEW

**Comment by maboiteaspam** [comment]
again, added debug message, but in another branch this time https://github.com/maboiteaspam/multicast-dns/tree/patch-1
to invoke like this `dns-discovery|bonjour [args] --verbose [dns-discovery|bonjour],multicast-dns` and see `multicast-dns` debug on demand. but not compatible 0.10 :/

**Comment by mafintosh** [comment]
thanks :)

File: example.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+var local = require('os').hostname() + '.local'
```
Comment (watson) [inline example.js]: @mafintosh on OS X the `os.hostname()` already includes the `.local` part - I'm not sure if this differs between operation systems, but I think the `+ '.local'` part should be removed

Comment (mafintosh) [inline example.js]: @watson could you fix this in master?

Comment (watson) [inline example.js]: @mafintosh fixed in 338acdb

**Pull request merged**